<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboard</class>
 <widget class="QMainWindow" name="AdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>925</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ŸÉÿßÿ™ÿ® ÿßŸÑÿ∂ÿ®ÿ∑</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#horizontalLayout{
background-color: #f3e8df;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#topBar{
background-color: #452829;
border-bottom: 3px solid white;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="notification">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"> #notification{
        background-color: transparent;
        font-size: 30px;
        color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255))    ;
    }
     #notification:hover{
		color: white;
    }</string>
         </property>
         <property name="text">
          <string>üîî</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="badge_label">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">
            background-color: red;
            color: white;
            border-radius: 50%;
            font-size: 10px;
            font-weight: bold;
          </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="fixedSize" stdset="0">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white; font-size: 30px; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>ŸÉÿßÿ™ŸÄŸÄŸÄÿ® ÿßŸÑÿ∂ŸÄÿ®ŸÄÿ∑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 50px; color: white;</string>
         </property>
         <property name="text">
          <string>üñå</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: transparent;</string>
         </property>
         <property name="text">
          <string>.....</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>40</number>
        </property>
        <property name="bottomMargin">
         <number>25</number>
        </property>
        <property name="horizontalSpacing">
         <number>35</number>
        </property>
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QStackedWidget" name="mainStack">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page_empty">
           <layout class="QVBoxLayout" name="verticalLayout_empty">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color: rgba(120, 120, 120, 0.6);</string>
              </property>
              <property name="text">
               <string>ŸÑÿß ŸäŸàÿ¨ÿØ ÿ®ŸäÿßŸÜÿßÿ™ ŸÑÿπÿ±ÿ∂Ÿáÿß !</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_documents">
           <layout class="QVBoxLayout" name="verticalLayout_docs">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">#scrollArea{
border: none;
}</string>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="files_widget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>1164</width>
                 <height>797</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="files_grid"/>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_master_record">
           <layout class="QVBoxLayout" name="verticalLayout_master">
            <item>
             <widget class="QLabel" name="label_master_title">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>ÿ≥ÿ¨ŸÑ ÿßŸÑÿ£ÿ≥ÿßÿ≥</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="masterRecordTable">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <column>
               <property name="text">
                <string>ÿ±ŸÇŸÖ ÿßŸÑŸÇÿ∂Ÿäÿ©</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ÿßÿ≥ŸÖ ÿßŸÑŸÖÿØÿπŸä</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ÿßÿ≥ŸÖ ÿßŸÑŸÖÿØÿπŸâ ÿπŸÑŸäŸá</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ŸÜŸàÿπ ÿßŸÑŸÇÿ∂Ÿäÿ©</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ÿ™ÿßÿ±ŸäÿÆ ÿßŸÑŸÖŸÑŸÅ</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ÿ≠ÿßŸÑÿ© ÿßŸÑŸÇÿ∂Ÿäÿ©</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_scheduling">
           <layout class="QVBoxLayout" name="verticalLayout_scheduling">
            <item>
             <widget class="QLabel" name="label_scheduling_title">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>ÿ¨ÿØŸàŸÑÿ© ÿ¨ŸÑÿ≥ÿ© ÿ¨ÿØŸäÿØÿ©</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="schedulingTable">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <column>
               <property name="text">
                <string>ÿßÿÆÿ™Ÿäÿßÿ±</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ÿ±ŸÇŸÖ ÿßŸÑŸÇÿ∂Ÿäÿ©</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ÿßÿ≥ŸÖ ÿßŸÑŸÖÿØÿπŸä</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ÿßÿ≥ŸÖ ÿßŸÑŸÖÿØÿπŸâ ÿπŸÑŸäŸá</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ŸÜŸàÿπ ÿßŸÑŸÇÿ∂Ÿäÿ©</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_scheduling">
              <item>
               <widget class="QLabel" name="label_judge">
                <property name="text">
                 <string>ÿßŸÑŸÇÿßÿ∂Ÿä:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="judgeComboBox">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_date">
                <property name="text">
                 <string>ÿ™ÿßÿ±ŸäÿÆ ÿßŸÑÿ¨ŸÑÿ≥ÿ©:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="sessionDateInput">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_time">
                <property name="text">
                 <string>ŸàŸÇÿ™ ÿßŸÑÿ¨ŸÑÿ≥ÿ©:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTimeEdit" name="sessionTimeInput"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btn_save_session">
              <property name="text">
               <string>ÿ≠ŸÅÿ∏ ÿßŸÑÿ¨ŸÑÿ≥ÿ©</string>
              </property>
              <property name="styleSheet">
               <string notr="true">
                background-color: #452829;
                color: white;
                font-size: 14px;
                padding: 10px;
                border-radius: 5px;
               </string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="sideBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #452829;
border: solid 20px #f3db93;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>25</number>
         </property>
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>22</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="add_case">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:#f3e8df;
border-bottom: 1px solid #f3e8df ;
padding-bottom: 10px;

}

QPushButton:hover {
    background: white;
	color:#452829;
	padding-bottom: 5px;
}</string>
           </property>
           <property name="text">
            <string>ÿ•ÿ∂ŸÄŸÄŸÄÿßŸÅÿ© ŸÇÿ∂Ÿäÿ© ‚ûï</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="master_record">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:#f3e8df;
border-bottom: 1px solid #f3e8df ;
padding-bottom: 10px;

}

QPushButton:hover {
    background: white;
	color:#452829;
	padding-bottom: 5px;
}</string>
           </property>
           <property name="text">
            <string>ÿ≥ÿ¨ŸÑ ÿßŸÑÿ£ÿ≥ÿßÿ≥ üóÑ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="case2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:#f3e8df;
border-bottom: 1px solid #f3e8df ;
padding-bottom: 10px;

}

QPushButton:hover {
    background: white;
	color:#452829;
	padding-bottom: 5px;
}</string>
           </property>
           <property name="text">
            <string>ÿ™ŸÇŸàŸäŸÖ ÿßŸÑŸÇÿßÿ∂Ÿä  üìÖ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_scheduling">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:#f3e8df;
border-bottom: 1px solid #f3e8df ;
padding-bottom: 10px;

}

QPushButton:hover {
    background: white;
	color:#452829;
	padding-bottom: 5px;
}</string>
           </property>
           <property name="text">
            <string>ÿ¨ÿØŸàŸÑÿ© ÿ¨ŸÑÿ≥ÿ© ÿ¨ÿØŸäÿØÿ© üìÖ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="docments">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:#f3e8df;
border-bottom: 1px solid #f3e8df ;
padding-bottom: 10px;

}

QPushButton:hover {
    background: white;
	color:#452829;
	padding-bottom: 5px;
}</string>
           </property>
           <property name="text">
            <string>ŸÑŸàÿßÿ¶ÿ≠ ÿßŸÑÿØÿπŸàŸâ üßæ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="logoutBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
color:black;
background-color: white;
border-radius: 10px;
}

QPushButton:hover {
    background: red;
	color:white;
}</string>
           </property>
           <property name="text">
            <string>ÿ™ÿ≥ÿ¨ŸäŸÑ ÿÆÿ±Ÿàÿ¨</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
